### DISCUSSION TITLE
Phaser v3.60.0

### DISCUSSION THREAD

**Discussion by photonstorm** [photonstorm/phaser#6469] [Announcements]
# Version 3.60.0 - Miku - 12th April 2023

Due to the size and importance of the v3.60 release, we have split the Change Log up into multiple sections.

This makes it easier for you to browse and find the information you need.

## New Features

These are the headliner features in this release:

* 14 bundled [Special FX](changelog/3.60/FX.md) including Bloom, Blur, Distort, Glow, Wipe, and more
* Vastly improved [Mobile Rendering Performance](changelog/3.60/MobilePerformance.md) - over 7000% faster!
* New [Timeline Sequencer](changelog/3.60/Timeline.md) for creating complex flows of events
* New [Plane Game Object](changelog/3.60/PlaneGameObject.md) for perspective distortions
* New [Nine Slice Game Object](changelog/3.60/NineSliceGameObject.md) for perfect UI scaling
* [Built-in Spector JS](changelog/3.60/Spector.md) for WebGL debugging on desktop and mobile
* Brand new [Video Game Object](changelog/3.60/VideoGameObject.md) handles videos and media streams with ease
* Brand new [Particle Emitter](changelog/3.60/ParticleEmitter.md) comes with explosive new features
* Support for [Spatial Audio](changelog/3.60/SpatialSound.md) and distance-based volume
* New [Spine 4 Plugin](changelog/3.60/Spine4.md) support
* Upgraded to [Matter Physics v0.19](changelog/3.60/MatterPhysics.md)
* New [Tween Manager](changelog/3.60/TweenManager.md) with better performance and memory management
* New [Dynamic Textures](changelog/3.60/DynamicTextures.md) for rendering to textures at runtime
* New [TimeStep features and Timer Event Updates](changelog/3.60/Timestep.md) for enforcing fps rates and more
* Support for [Compressed Textures](changelog/3.60/CompressedTextures.md)
* [ESM Module Support](changelog/3.60/ESMSupport.md)

## System and Plugins

Pick any of the following sections to see the breaking changes, new features, updates, and bug fixes for that area of the API.

* [Animation System](changelog/3.60/Animation.md)
* [Arcade Physics](changelog/3.60/ArcadePhysics.md)
* [Bitmap and Geometry Masks](changelog/3.60/Masks.md)
* [Camera System](changelog/3.60/Camera.md)
* [Canvas Renderer](changelog/3.60/CanvasRenderer.md)
* [WebGL Renderer](changelog/3.60/WebGLRenderer.md)
* [Colors and Display](changelog/3.60/Colors.md)
* [Game, Device and Game Config](changelog/3.60/Game.md)
* [Geometry, Paths and Curves](changelog/3.60/Geometry.md)
* [Input System](changelog/3.60/Input.md)
* [Loader System](changelog/3.60/Loader.md)
* [Scale Manager](changelog/3.60/ScaleManager.md)
* [Scenes and Scene Manager](changelog/3.60/Scene.md)
* [Sound System](changelog/3.60/Sound.md)
* [Spine 3 Plugin](changelog/3.60/Spine3.md)
* [Texture Manager](changelog/3.60/TextureManager.md)
* [Utils, Math and Actions](changelog/3.60/Utils.md)
* [Build Config and Browser Updates](changelog/3.60/Build.md)

## Game Object Updates

Finally, here are the updates related to Game Objects:

* [Bitmap Text Game Object](changelog/3.60/BitmapTextGameObject.md)
* [Container Game Object](changelog/3.60/Container.md)
* [Graphics Game Object](changelog/3.60/GraphicsGameObject.md)
* [Mesh Game Object, Vertices and Faces](changelog/3.60/Mesh.md)
* [Text Game Object](changelog/3.60/TextGameObject.md)
* [Tilemap Game Object](changelog/3.60/Tilemap.md)
* [All other Game Object related Updates](changelog/3.60/GameObject.md)

## Examples, Documentation, Beta Testing and TypeScript

My thanks to the following for helping with the Phaser 3 Examples, Beta Testing, Docs, and TypeScript definitions, either by reporting errors, fixing them, or helping author the docs:

| 💖 | 💖 | 💖 | 💖 |
| ----- | ----- | ----- | ----- |
| @0day-oni | @201flaviosilva | @AlbertMontagutCasero | @Arcanorum |
| @arosemena | @austinlyon | @chrisl8 | @christian-post |
| @danfoster | @darrylpizarro | @DeweyHur | @drunkcat |
| @ef4 | @eltociear | @EsteFilipe | @etherealmachine |
| @EmilSV | @Fake | @florestankorp | @hacheraw |
| @hanzooo | @jerricko | @joegaffey | @jonasrundberg |
| @kainage | @kootoopas | @lolimay | @MaffDev |
| @michalfialadev | @monteiz | @necrokot | @Nero0 |
| @OdinvonDoom | @orjandh | @pavle-goloskokovic | @PhaserEditor2D |
| @Pythux | @quocsinh | @rgk | @rollinsafary-inomma |
| @rstanuwijaya | 👑 @samme 👑 | @Smirnov48 | @steveja42 |
| @sylvainpolletvillard | @twoco | @ubershmekel | @ultimoistante |
| @VanaMartin | @vforsh | @Vidminas | @x-wk |
| @xmahle | @xuxucode | @YeloPartyHat | @ZekeLu |
| FromChris | Golen | OmniOwl | and you ... |

📖 Read the [Phaser 3 API Docs](https://newdocs.phaser.io/) 💻 Browse 2000+ [Code Examples](https://labs.phaser.io) 🤝 Join the awesome [Phaser Discord](https://discord.gg/phaser)

<hr /><em>This discussion was created from the release <a href='https://github.com/photonstorm/phaser/releases/tag/v3.60.0'>Phaser v3.60.0</a>.</em>

**Comment by ChildLearningClub** [comment]
Thank you
